spec/requests/api/v0/classified_listings_spec.rb

```diff
@@ -60,18 +56,18 @@ def json_response
 
     it "returns listings created" do
       get api_classified_listings_path
-      expect(json_response.size).to eq(7)
-      expect(json_response.first["type_of"]).to eq("classified_listing")
-      expect(json_response.first["slug"]).to eq(ClassifiedListing.last.slug)
-      expect(json_response.first["user"]).to include("username")
-      expect(json_response.first["user"]["username"]).not_to be_empty
+      expect(response.parsed_body.size).to eq(7)
```

Comment:
`parsed_body` is seriously one of my new favorite shortcuts!

Reply:
ahah learned it from you :D